sample.go

```diff
@@ -654,18 +680,17 @@ func (s *UniformSample) Sum() int64 {
 }
 
 // Update samples a new value.
-func (s *UniformSample) Updates(vv []int64)  {
-	for _,v := range vv {
+func (s *UniformSample) Updates(vv []int64) {
+	for _, v := range vv {
 		s.updateInner(v)
 	}
 }
 
 // Update samples a new value.
-func (s *UniformSample) Update(v int64) float64 {
+func (s *UniformSample) Update(v int64) {
 	s.mutex.Lock()
```

Comment:
hrm, thinking we should probably make this lib not use locks and call that out in our code that you do the locking yourself. this will be far faster.